cms/templates/course_outline.html

```diff
@@ -114,6 +115,47 @@ <h2 class="title title-3 warning-heading-text">${_("This course uses features th
         </div>
       </div>
     %endif
+  
+  %if proctoring_errors:
+    <div class="wrapper wrapper-alert wrapper-alert-error is-shown">
+      <div class="alert announcement has-actions">
+        <span class="feedback-symbol fa fa-warning" aria-hidden="true"></span>
+
+        <div class="copy">
+          <h2 class="title title-3">${_("This course has proctored exam settings that are incomplete or invalid.")}</h2>
+          <div class="errors-list">
+            <p>
+              % if course_authoring_microfrontend_url:
+                <% url_encoded_course_id = quote(six.text_type(context_course.id).encode('utf-8'), safe='') %>
+                ${Text(_("To update these settings go to the {link_start}Exam Settings page{link_end}.")).format(
```

Comment:
nit: I think it might be better to use `Proctored Exam Settings` here, because that is the name a user will find under the settings drop down